tests/testutils/kubeutils/kind_cluster.go

```diff
@@ -171,9 +171,18 @@ func (k KindCluster) runKubectl(stdin io.Reader, args ...string) (stdOut, stdErr
 		// panic here to prevent swallowing what would have been a fatal error
 		panic(err)
 	})
+
+	defer func() {
+		// recovered from fatal kubectl command
+		if e := recover(); e != nil {
+			err = multierr.Combine(err, e.(error))
```

Comment:
Is that cast safe?

Reply:
Not if the panic originated elsewhere w/o using an error value. Updated.